src/instana/util/traceutils.py

```diff
@@ -64,4 +92,16 @@ def get_tracer_tuple() -> (
 
 
 def tracing_is_off() -> bool:
+    """Check if tracing is currently disabled."""
     return not (bool(get_active_tracer()) or agent.options.allow_exit_as_root)
+
+
+def check_if_ignored(
+    instrumentation: str,
+    command: str,
+) -> bool:
+    """Check if the given instrumentation and command combination should be ignored."""
+    return (
+        instrumentation in agent.options.ignore_endpoints
+        or f"{instrumentation}.{command.lower()}" in agent.options.ignore_endpoints
+    )
```

Comment:
A better name for this method is `is_service_or_endpoint_ignored`; the variable names must be coherent.
```suggestion
def is_service_or_endpoint_ignored(
    service: str,
    endpoint: str,
) -> bool:
    """Check if the given service and endpoint combination should be ignored."""
    return (
        service in agent.options.ignore_endpoints
        or f"{service}.{endpoint.lower()}" in agent.options.ignore_endpoints
    )
```